examples/js/loaders/LDrawLoader.js

```diff
@@ -469,6 +469,50 @@ THREE.LDrawLoader = ( function () {
 
 		}
 
+		if ( isConditionalSegments ) {
```

Comment:
Should this go inside the `elementSize === 2` condition above because it's only relevant for lines? It'll work either way just something I noticed

Reply:
Yep, there is an implicit invariant (restriction) for the parameters: `! isConditionalSegments || elementSize === 2`.
Just have to keep in mind this for a future hypothetic refactoring.